chsdi/templates/htmlpopup/cadastralwebmap_opendata.mako

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
```

Comment:
i have tried to use inherit but i did not make it. so for now it is done with an include.